tests/integration/security/external_ca/main_test.go

```diff
@@ -81,9 +81,7 @@ func TestMain(m *testing.M) {
 	// nolint: staticcheck
 	framework.NewSuite(m).
 		Label(label.CustomSetup).
-		RequireMinVersion(19).
-		RequireSingleCluster().
-		RequireMultiPrimary().
+		RequireMinVersion(20).
```

Comment:
just curious why this changed?

Reply:
Changed it for testing, will revert it to the original version.